sql/catalyst/src/main/scala/org/apache/spark/sql/types/Decimal.scala

```diff
@@ -17,9 +17,10 @@
 
 package org.apache.spark.sql.types
 
-import java.math.{RoundingMode, MathContext}
+import java.math.{MathContext, RoundingMode, BigDecimal => JavaBigDecimal}
 
 import org.apache.spark.annotation.DeveloperApi
+import org.apache.spark.unsafe.PlatformDependent
 
 /**
  * A mutable implementation of BigDecimal that can hold a Long if values are small enough.
```

Comment:
This comment is out of date now that we're relying on Java's `BigDecimal` to manage the compact representation for us, right?